usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -255,6 +255,7 @@ typedef struct {
     size_t control_points_len;
     int strict_mode;
     int vhsm_mode;
+    int optimize_single_ops;
```

Comment:
i feel "one-shot" singn/verify resp. enc/dec is commonly used to decribe non-init-update-final apis. This applies to all namings of functions variable etc. However, calling it "single op" would still be okay for me since its internal names only.

Reply:
The EP11 library functions uses the term "single", as in m_SignSingle().